### ISSUE TITLE
Can't resolve e2mmap/thwait gems when generating on Ruby 2.6.5

### ISSUE THREAD

**Issue by tallytarik** [Shopify/tapioca#39]
[This change](https://github.com/Shopify/tapioca/commit/3831e5971bfb36d336f3331d3b90aad0682c2575) in 0.2.8 causes an error when trying to run `bundle exec tapioca generate` on Ruby 2.6.5:

```
Errno::ENOENT: No such file or directory @ realpath_rec - /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thwait-0.1.0
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca/gemfile.rb:92:in `realpath'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca/gemfile.rb:92:in `initialize'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca/gemfile.rb:36:in `new'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca/gemfile.rb:36:in `block in dependencies'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca/gemfile.rb:36:in `map'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca/gemfile.rb:36:in `dependencies'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/sorbet-runtime-0.5.5460/lib/types/private/methods/_methods.rb:240:in `call'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/sorbet-runtime-0.5.5460/lib/types/private/methods/_methods.rb:240:in `block in _on_method_added'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca/generator.rb:254:in `gems_to_generate'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/sorbet-runtime-0.5.5460/lib/types/private/methods/_methods.rb:240:in `call'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/sorbet-runtime-0.5.5460/lib/types/private/methods/_methods.rb:240:in `block in _on_method_added'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca/generator.rb:58:in `build_gem_rbis'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/sorbet-runtime-0.5.5460/lib/types/private/methods/_methods.rb:240:in `call'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/sorbet-runtime-0.5.5460/lib/types/private/methods/_methods.rb:240:in `block in _on_method_added'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca/cli.rb:56:in `block in generate'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca.rb:10:in `silence_warnings'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/lib/tapioca/cli.rb:55:in `generate'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/command.rb:27:in `run'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/invocation.rb:127:in `invoke_command'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor.rb:392:in `dispatch'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/base.rb:485:in `start'
  /Users/me/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/tapioca-0.2.8/exe/tapioca:6:in `<top (required)>'
  /Users/me/.rbenv/versions/2.6.5/bin/tapioca:23:in `load'
  /Users/me/.rbenv/versions/2.6.5/bin/tapioca:23:in `<top (required)>'
```

You can replicate by adding e.g. `gem 'sidekiq-scheduler'` to your Gemfile, which depends on `thwait` and `e2mmap`.

These gems in Ruby 2.6.5, so they're not installed by bundler. The path that tapioca tries to load from, above, is wrong.

The gems are actually located under `lib/ruby/2.6.0`. In my case that's `/Users/me/.rbenv/versions/2.6.5/lib/ruby/2.6.0/thwait`

Version 0.2.7 works as expected.

**Comment by paracycle** [comment]
@tallytarik Indeed, that issue was raised internally as well and was fixed in #40. I'll make another release with that shortly.

Thanks for reporting and sorry about the breaking change.

**Comment by paracycle** [comment]
I just released this in version 0.3.0
